src/main/resources/org/jenkinsci/plugins/saml/SamlEncryptionData/config.jelly

```diff
@@ -18,4 +18,7 @@
     <f:entry title="Auth Request Signature" field="forceSignRedirectBindingAuthnRequest" help="/plugin/saml/help/forceSignRedirectBindingAuthnRequest.html">
         <f:checkbox/>
     </f:entry>
+    <f:entry title="Wants Assertion Signed" field="wantsAssertionsSigned" help="/plugin/saml/help/wantsAssertionsSigned.html">
```

Comment:
BTW you can
```suggestion
    <f:entry title="Wants Assertion Signed" field="wantsAssertionsSigned">
```
if you just
```bash
git mv src/main/webapp/help/wantsAssertionsSigned.html src/main/resources/org/jenkinsci/plugins/saml/SamlEncryptionData/help-wantsAssertionsSigned.html
```

Reply:
I know, but the help files were in a separate folder since I grab the plugin and sincerely I like it.